setup.cfg

```diff
@@ -1,3 +1,24 @@
+[bumpversion]
+current_version = 4.0.dev0
+commit = True
+tag = True
+tag_name=yt-{new_version}
+serialize = {major}.{minor}.dev{patch}
+parse = (?P<major>\d+)\.(?P<minor>\d+)\.dev(?P<patch>\d+)
```

Comment:
If I'm following everything, you're plan is to manually remove the `dev` part each time we're about to perform a release, but this feels a bit anticlimactic as this work is about automation.
From `bump2version`'s doc, it seems to me that a solution can be elaborated from their `optional_value =` section with the `<release>` variable, though I don't have time to build this myself right now.

Reply:
Ok, I *think* I understand it from reading this [blogpost](https://williamhayes.medium.com/versioning-using-bumpversion-4d13c914e9b8) and I'll give a shot at that...